app/javascript/schools/founders/components/students_panel/SA_StudentsPanel_UpdateDetailsForm.re

```diff
@@ -0,0 +1,393 @@
+          }
+          className="appearance-none block w-full bg-white border border-gray-400 rounded py-3 px-4 leading-snug focus:outline-none focus:bg-white focus:border-gray-500"
+          id="affiliation"
+          type_="text"
+          placeholder="Acme Inc., Acme University, etc."
+        />
+      </div>
+      <div className="mt-5">
+        <div className="border-b pb-4 mb-2 mt-5 ">
+          <span className="inline-block mr-1 text-xs font-semibold">
+            {(isSingleFounder ? "Personal Coaches" : "Team Coaches") |> str}
+          </span>
+          <div className="mt-2">
+            <School__SelectBox
+              items={state.teamCoaches}
+              selectCB=multiSelectCoachEnrollmentsCB
+            />
+          </div>
+        </div>
+      </div>
+      <div className="mt-5">
+        <div className="mb-2 text-xs font-semibold">
+          {"Tags applied:" |> str}
+        </div>
+        <SA_StudentsPanel_SearchableTagList
+          unselectedTags={
+            studentTags
+            |> List.filter(tag => !(state.tagsToApply |> List.mem(tag)))
+          }
+          selectedTags={state.tagsToApply}
+          addTagCB={tag => send(AddTag(tag))}
+          removeTagCB={tag => send(RemoveTag(tag))}
+          allowNewTags=true
+        />
+      </div>
+      <div className="mt-5">
+        <div className="flex items-center flex-shrink-0">
+          <label className="block tracking-wide text-xs font-semibold mr-3">
+            {"Should this student be excluded from leaderboards?" |> str}
+          </label>
+          <div
+            className="flex flex-shrink-0 rounded-lg overflow-hidden border border-gray-400">
+            <button
+              title="Exclude this student from the leaderboard"
+              onClick={event => {
+                ReactEvent.Mouse.preventDefault(event);
+                send(UpdateExcludedFromLeaderboard(true));
+              }}
+              className={boolBtnClasses(state.excludedFromLeaderboard)}>
+              {"Yes" |> str}
+            </button>
+            <button
+              title="Include this student in the leaderboard"
+              onClick={_event => {
+                ReactEvent.Mouse.preventDefault(_event);
+                send(UpdateExcludedFromLeaderboard(false));
+              }}
+              className={boolBtnClasses(!state.excludedFromLeaderboard)}>
+              {"No" |> str}
+            </button>
+          </div>
+        </div>
+      </div>
+      <div className="mt-5">
+        <label
+          className="tracking-wide text-xs font-semibold"
+          htmlFor="access-ends-at-input">
+          {"Access ends at" |> str}
+        </label>
+        <span className="ml-1 text-xs"> {"(optional)" |> str} </span>
+        <HelpIcon
+          className="ml-2"
+          link="https://docs.pupilfirst.com/#/students?id=editing-student-details">
+          {"If specified, students can't submit their work from the specified date"
```

Comment:
@bodhish: It's good to tweak the message so that it's clear changing `access_ends_at` affects the whole team when the team size is more than 1.